aten/src/ATen/native/native_functions.yaml

```diff
@@ -13131,12 +13132,6 @@
     NestedTensorCPU: NestedTensor_softmax_dropout
     NestedTensorCUDA: NestedTensor_softmax_dropout_cuda
 
-- func: _nested_tensor_layer_norm(Tensor self, Tensor? weight, Tensor? bias, float eps) -> Tensor
```

Comment:
Going to have to update the backward compat probably

Reply:
is this just documentation marking the function as gone or are there extra steps?

Reply:
A test failure will likely pop up for from here: https://github.com/pytorch/pytorch/blob/master/test/forward_backward_compatibility/check_forward_backward_compatibility.py